packages/middleware-sdk-route53/src/change-batch-alias-target-id-normalizer.ts

```diff
@@ -0,0 +1,86 @@
+import {
+  InitializeHandler,
+  InitializeMiddleware,
+  InitializeHandlerArguments,
+  InitializeHandlerOptions,
+  InitializeHandlerOutput,
+  MetadataBearer,
+  Pluggable
+} from "@aws-sdk/types";
+import { IDENTIFIER_PREFIX_PATTERN } from "./constants";
+
+export interface Change {
+  ResourceRecordSet: {
+    AliasTarget?: {
+      HostedZoneId: string;
+    };
+  };
+}
+
+export interface ChangeBatchBearer {
+  ChangeBatch: {
+    Changes: Iterable<Change>;
+  };
+}
+
+export function changeBatchAliasTargetIdNormalizerMiddleware(): InitializeMiddleware<
```

Comment:
What does `normalizer` mean in the middleware name?

Reply:
It changes that target using the same prefix pattern as the idNormalizerMiddleware